client/app/lib/getQueryResultData.js

```diff
@@ -1,6 +1,7 @@
+import { useMemo } from "react";
 import { get, invoke } from "lodash";
 
-export default function getQueryResultData(queryResult) {
+export function getQueryResultData(queryResult) {
```

Comment:
Do we still use getQueryResultData anywhere? If not, let's change useQueryResultData to be the default export and rename the file.